include/apps/tools/OswAppToolsApp.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <osw_hal.h>
+#include <osw_ui.h>
+#include "osw_app.h"
+
+class OswAppToolsApp : public OswApp {
```

Comment:
This is not (really) needed, because... See comment in main file.

Reply:
ok